scalyr_agent/monitors_manager.py

```diff
@@ -369,7 +370,13 @@ def __start_monitor(self, monitor):
         # Check to see if we can open the metric log.  Maybe we should not silently fail here but instead fail.
         if monitor.open_metric_log():
             monitor.config_from_monitors(self)
-            log.info("Starting monitor %s", monitor.monitor_name)
+            starting_minitor_message = "Starting monitor %s." % monitor.monitor_name
+            if monitor.get_stop_agent_on_failure():
+                starting_minitor_message += (
+                    " Config option 'stop_agent_on_failure' is enabled."
```

Comment:
Would you mind sharing output of status -v + agent.info start up log with this change included? Thanks.

Reply:
ah, sure.

Reply:
@Kami updated main PR description.